### PR TITLE
Page title

### PR DESCRIPTION
**Pull request by ghost**
## Summary

- page-title

## Screenshots

Captura de pantalla de 2019-05-21 17-33-26

## Trello Card

https://trello.com/c/Me3n8eDC/21-page-titles

### CODE REVIEW

File: maquetado/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+	<h2 class="title">Front-End Tools</h2>
```
Comment (mmamaniwolox) [inline maquetado/index.html]: Check indentation

File: maquetado/scss/application.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@import 'colors';
+@import 'equalizer';
+@import 'titles';
+
```
Comment (mmamaniwolox) [inline maquetado/scss/application.scss]: You have 2 extra lines here

File: maquetado/scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,32 @@
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+// .title {
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: remove commented code

File: maquetado/scss/titles.scss [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  font-size: $text-big;
+  margin: 20px;
+  position: relative;
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;
+
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: delete this empty line

File: maquetado/scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;
+    
+    width: 100px;
+    height: 100px;
+    display: block;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: remove this

File: maquetado/scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;
+    
+    width: 100px;
+    height: 100px;
+    display: block;    
+    position: absolute;   
+    right: 220px;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: try with a lower `left` value

File: maquetado/scss/titles.scss [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    top: 56px;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: this shouldnt be so high

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]
Lets order the css elements inside a class alphabetically!

File: maquetado/scss/titles.scss [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+.title {
+  font-size: $text-big;
+  margin: 20px;
+  position: relative;
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;
```
Comment (Dorian30) [inline maquetado/scss/titles.scss]: You shouldn't need this.

**Review by Dorian30 — CHANGES_REQUESTED** [review]

File: maquetado/index.html [hunk 9]
```diff
@@ -9,6 +9,6 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+	<h2 class="title">Front-End Tools</h2>
```
Comment (mmamaniwolox) [inline maquetado/index.html]: indentation here too

File: maquetado/scss/titles.scss [hunk 10]
```diff
@@ -0,0 +1,25 @@
+$text-big: 24px;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: All this text styles sizes could be in a `fonts.scss` file. (not the `title-decoration` ones)

File: maquetado/scss/application.scss [hunk 11]
```diff
@@ -1,6 +1,3 @@
 @import 'colors';
 @import 'equalizer';
-
-.title {
-	color: $red;
-}
+@import 'titles';
```
Comment (mmamaniwolox) [inline maquetado/scss/application.scss]: You are missing the font-family style, you can add it to this file this way

```
* {
  font-family: 'Roboto', sans-serif;
}
```

File: maquetado/scss/titles.scss [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+  font-size: $text-big;
+  margin: 20px;
+  position: relative;
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;    
+		display: block;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: you have some indentation issues here. I think you dont need the `display: block` prop

File: maquetado/scss/titles.scss [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+.title {
+  font-size: $text-big;
+  margin: 20px;
+  position: relative;
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;
```
Comment (mmamaniwolox) [inline maquetado/scss/titles.scss]: Do you want it to repeat the image vertically?

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/scss/fonts.scss [hunk 14]
```diff
@@ -0,0 +1,5 @@
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
```
Comment (Dorian30) [inline maquetado/scss/fonts.scss]: Add an empty line here

File: maquetado/scss/titles.scss [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+  font-size: $text-big;
+  margin: 20px;
+  position: relative;
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: no-repeat;    
+		height: 100px;
```
Comment (Dorian30) [inline maquetado/scss/titles.scss]: Check this. You have a lot of white space here.

**Review by Dorian30 — CHANGES_REQUESTED** [review]

**Review by Dorian30 — APPROVED** [review]

File: maquetado/scss/fonts.scss [hunk 16]
```diff
@@ -0,0 +1,5 @@
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$title-decoration-height: 90px;
```
Comment (mmamaniwolox) [inline maquetado/scss/fonts.scss]: This is not a font constant, more a specific one. I would put this in the same file you styled the title element

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**